python/pyspark/sql/udf.py

```diff
@@ -181,3 +183,180 @@ def asNondeterministic(self):
         """
         self.deterministic = False
         return self
+
+
+class UDFRegistration(object):
```

Comment:
This seems introduced from 1.3.1 - https://issues.apache.org/jira/browse/SPARK-6603